### PR TITLE
Translate documents in Korean (01,02,03,04) #138

### LINKED ISSUE #138: [Tutorial/Basic] Create Korean documents, 01 - 04
Create tutorial documents in Korean

- 01 getting-started-kr.md
- 02 change-background-and-title-kr.md
- 03 cardview-kr.md
- 04 scrollview-kr.md

### PR DESCRIPTION
**Pull request by sonypark**
resolve #138 

## Fix
- Add missing image on a post
- Add Github source link

## Changes
- Create tutorial documents in Korean
- 01 getting-started-kr.md
- 02 change-background-and-title-kr.md
- 03 cardview-kr.md
- 04 scrollview-kr.md

#### (Automated Tests)
- N/A

#### (Manual Tests)
- Test in local with jekyll

### Notes
- Request review on this PR to @Haerang94

### References
- https://facebook.github.io/react-native/docs/getting-started

### CODE REVIEW

File: docs/basic-tutorial/basic-features(todolist)/01-getting-started/getting-started-kr.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+튜토리얼을 시작하기 전에 개발 환경 설정을 하도록 하겠습니다.
+
+환경설정 방법은 React Native 공식 홈페이지에 자세히 나와있습니다. 아래 링크를 참고하셔서 설정하시면 됩니다.
+
+- [https://facebook.github.io/react-native/docs/0.60/getting-started](https://facebook.github.io/react-native/docs/0.60/getting-started)
+
+### Create New Project
+
+환경 설정을 모두 마치셨으면, React Native CLI 명령어로 React Native 프로젝트를 만들어보겠습니다.
```
Comment (qkreltms) [inline docs/basic-tutorial/basic-features(todolist)/01-getting-started/getting-started-kr.md]: "환경 설정을 모두 완료했다면, " This one would be better for me.

File: docs/basic-tutorial/basic-features(todolist)/01-getting-started/getting-started-kr.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+parent: 할 일 목록 앱 만들기
+grand_parent: Basic
+nav_order: 1
+---
+
+## 튜토리얼 소개
+
+React Native를 사용하여 간단한 할 일 목록 애플리케이션을 만드는 튜토리얼 입니다. React Native로 만든 애플리케이션은 기본적으로 iOS와 안드로이드 플랫폼에서 모두 동작합니다. 하지만 iOS와 안드로이드 환경 간 레이아웃이나 기타 설정 등에 차이가 있을 수 있습니다. 이번 튜토리얼에서는 iOS를 기본 개발 환경으로 두고 진행합니다.
+
+React Native 애플리케이션을 만드는 방법으로는 Expo 또는 React Native CLI를 사용할 수 있습니다. Expo를 사용하면 쉽게 프로젝트를 설치하고 실행할 수 있는 장점이 있지만 개발 환경이 Expo에서 제공하는 기능에 한정되어 Native 환경을 모두 이용할 수는 없다는 단점이 있습니다. React Native CLI를 이용하면 설치와 실행이 Expo보다는 다소 까다롭지만 Native 모듈에서 제공하는 기능을 모두 이용할 수 있다는 장점이 있습니다. 본 튜토리얼에서는 React Native CLI를 이용해 진행하겠습니다.
```
Comment (qkreltms) [inline docs/basic-tutorial/basic-features(todolist)/01-getting-started/getting-started-kr.md:13]: 간결하게 "본 튜토리얼에서는 React Native CLI를 이용해 진행하겠습니다" 만 적어도 괜찮을 것 같습니다.

**Review by qkreltms — CHANGES_REQUESTED** [review]

File: docs/basic-tutorial/basic-features(todolist)/01-getting-started/getting-started-kr.md (hunk 2 above)
Comment (JeffGuKang) [reply docs/basic-tutorial/basic-features(todolist)/01-getting-started/getting-started-kr.md:13]: I think this is OK.

File: docs/basic-tutorial/basic-features(todolist)/01-getting-started/getting-started-kr.md (hunk 1 above)
Comment (JeffGuKang) [reply docs/basic-tutorial/basic-features(todolist)/01-getting-started/getting-started-kr.md]: > 
> 
> "환경 설정을 모두 완료했다면, " This one would be better for me.

I agree with this

Comment (sonypark) [reply docs/basic-tutorial/basic-features(todolist)/01-getting-started/getting-started-kr.md]: That's right, Thanks for your suggestion.

**Review by JeffGuKang — APPROVED** [review]

**Pull request merged**